python/ml4ir/base/model/architectures/dnn.py

```diff
@@ -21,7 +23,12 @@ class DNNLayer:
 class DNN:
     def __init__(self, model_config: dict, feature_config: FeatureConfig, file_io):
         self.file_io: FileIO = file_io
+        self.model_config = model_config
+        self.feature_config = feature_config
         self.layer_ops: List = self.define_architecture(model_config, feature_config)
+        if 'positional_bias_handler' in self.model_config and self.model_config['positional_bias_handler'][
+            'key'] == PositionalBiasHandler.FIXED_ADDITIVE_POSITIONAL_BIAS:
+            self.positional_bias_layer = FixedAdditivePositionalBias()
```

Comment:
@mohazahran For the future, it might be better to create a separate factory method that does this. Will be decoupled from DNN implementation and will have a neat lookup table for all positional bias implementations.

Reply:
It's okay to not do this for now though I think. But make sure you nest the if instead of and here.